src/app/my/reserve-calendar/page.tsx

```
@@ -8,13 +8,14 @@ import { getActListApi } from '@/features/activities/libs/api/getActListApi';
 import { getReservationsByMonthApi } from '@/features/activities/libs/api/getReserveMonthApi';
 import { useAuthStore } from '@/features/auth/stores/useAuthStore';
-import { ContentReservation } from '@/features/reservation-state/components/content-reservation';
-import EmptyReservation from '@/features/reservation-state/components/empty-reservation';
 import CalendarWithReservations from '@/shared/components/calendar/components/calendar-with-reservations';
 import { MonthReservations } from '@/shared/components/calendar/libs/types/data';
 import Dropdown from '@/shared/components/dropdown';
 import AdaptiveModal from '@/shared/components/modal/components/adaptive-modal/adaptive-modal';
+import { ContentReservation } from '@/shared/components/modal/components/adaptive-modal/content-reservation';
+import EmptyReservation from '@/shared/components/modal/components/adaptive-modal/empty-reservation';
 import { useModalStore } from '@/shared/components/modal/libs/stores/useModalStore';
+import useWindowSize from '@/shared/libs/hooks/useWindowSize';
```

ESLint

[error] 'useWindowSize' is defined but never used.

(unused-imports/no-unused-imports)